Description: Upstream changes introduced in version 5.5-3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 pymecavideo (5.5-3) unstable; urgency=low
 .
   * changed the build-dependency rsvg -> librsvg2-bin. Closes: #629732
 .
 The person named in the Author field signed this changelog entry.
Author: Georges Khaznadar <georgesk@ofset.org>
Bug-Debian: http://bugs.debian.org/629732

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- pymecavideo-5.5.orig/src/globdef.py
+++ pymecavideo-5.5/src/globdef.py
@@ -41,7 +41,7 @@ licence['fr']=u"""
 """
 
 
-import sys, os
+import sys, os, os.path, subprocess
 from PyQt4.QtGui import QDesktopServices
 #from PyQt4.QtGui import *
 
@@ -95,6 +95,8 @@ if sys.platform == 'win32':
 else:
     datalocation=os.path.join("%s"%QDesktopServices.storageLocation(QDesktopServices.DataLocation),"pymecavideo")
     PATH = APP_DATA_PATH = datalocation
+    if not os.path.exists(datalocation):
+        subprocess.call("mkdir -p %s" %datalocation, shell=True)
 
 
 #
